custom-commands/experimental/custom-commands-offline/Run-CC-Offline.md

```diff
@@ -0,0 +1,123 @@
+# Running the Custom Command Embedded Service on Windows
+
+## Prerequisites
+
+### The host computer
+
+The host is an x-64 or arm-64 based computer that runs the Docker container.

```

Comment:
could we add information about recommended space, memory and compute?

Reply:
Just noticed it below. If we could structure all the pre-reqs together that will be awesome!